webpages/settings/index.html

```diff
@@ -13,6 +13,7 @@
     <div class="navbar">
       <img src="../../images/icon.svg" class="logo" alt="Logo" />
       <h1>Settings</h1>
+      <img @click="switchTheme()" class="theme-switch" src="../../images/icons/theme.svg" />
```

Comment:
Please do `setTheme(!theme)`, this function is not necessary.
Also, can you change the icon accordingly? The website does that

Reply:
Whoops! I forgot about that! Yeah, I can have it do that.